### PR TITLE
Signing artifacts and updated release notes to have steps to be followed for installation and verification

### PR DESCRIPTION
**Pull request by kumaritanushree**
### **With this change release notes will look like:**

 # :sparkles: What's new
* Test PR by @kumaritanushree in https://github.com/kumaritanushree/secretgen-controller/pull/1

## New Contributors
* @kumaritanushree made their first contribution in https://github.com/kumaritanushree/secretgen-controller/pull/1

<details>
<summary><h2>Installation and signature verification</h2></summary>

## Verify checksums file signature

Install cosign on your system https://docs.sigstore.dev/system_config/installation/

The checksums file provided within the artifacts attached to this release is signed using [Cosign](https://docs.sigstore.dev/cosign/overview/) with GitHub OIDC. To validate the signature of this file, run the following commands:

```shell
# Download the checksums file, certificate, and signature
curl -LO https://github.com/kumaritanushree/secretgen-controller/releases/download/v0.0.15/checksums.txt
curl -LO https://github.com/kumaritanushree/secretgen-controller/releases/download/v0.0.15/checksums.txt.pem
curl -LO https://github.com/kumaritanushree/secretgen-controller/releases/download/v0.0.15/checksums.txt.sig

### Verify the checksums file
cosign verify-blob checksums.txt --certificate checksums.txt.pem --signature checksums.txt.sig --certificate-identity-regexp=https://github.com/${{ github.repository_owner }} --certificate-oidc-issuer=https://token.actions.githubusercontent.com 
```

## Installation of secretgen-controller

secretgen-controller can be installed by using kapp 
```shell
kapp deploy -a kc -f https://github.com/kumaritanushree/secretgen-controller/releases/v0.0.15/download/release.yml
```

or by using kubectl
```shell
kubectl deploy -f https://github.com/kumaritanushree/secretgen-controller/releases/v0.0.15/download/release.yml
```

### Container Images

secretgen-controller and secretgen-controller-package-bundle images are available in Github Container Registry.

### OCI Image URLs

- ghcr.io/kumaritanushree/secretgen-controller@sha256:70a103b38704d50f1dae1d7b63a2e0cd49b8e4a639de4e283414f2a6c6a1da8d
- ghcr.io/kumaritanushree/secretgen-controller-package-bundle@sha256:de320920cc33a56620bfb7a54e49da169037f7aa9c5d6538edce3b0d3b242231

### Verify container image signature

The container images are signed using [Cosign](https://docs.sigstore.dev/cosign/overview/) with GitHub OIDC. To validate the signature of OCI images, run the following commands:
```shell
# Verifying secretgen-controller image
cosign verify ghcr.io/kumaritanushree/secretgen-controller@sha256:70a103b38704d50f1dae1d7b63a2e0cd49b8e4a639de4e283414f2a6c6a1da8d --certificate-identity-regexp=https://github.com/kumaritanushree --certificate-oidc-issuer=https://token.actions.githubusercontent.com -o text

# Verifying secretgen-controller-package-bundle image
cosign verify ghcr.io/kumaritanushree/secretgen-controller-package-bundle@sha256:de320920cc33a56620bfb7a54e49da169037f7aa9c5d6538edce3b0d3b242231 --certificate-identity-regexp=https://github.com/kumaritanushree --certificate-oidc-issuer=https://token.actions.githubusercontent.com -o text
```

</summary>
</details>

**Full Changelog**: https://github.com/kumaritanushree/secretgen-controller/compare/v0.0.13...v0.0.15

# :open_file_folder: Files Checksum
```
9228b98307ed972515110a14219a183930e35edb992298b23885b17b47bcd1cb  ./release.yml
ba288be541519d6b6f38ad7b5a9e8d958216fe8c6e5fdf638d5849a6ede56b11  ./package.yml
8f185deaa02964663bc3cedcb7c0af46151676e7c2abc2b69ff04178f87f28fe  ./package-metadata.yml
```

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -42,23 +48,165 @@ jobs:
...
+        run: |
+          image_url=`yq e '.spec.template.spec.fetch[0].imgpkgBundle.image' release/package.yml`
+          cosign verify \
+            $image_url \
+            --certificate-identity-regexp=https://github.com/${{ github.repository_owner }} \
+            --certificate-oidc-issuer=https://token.actions.githubusercontent.com
+
+      - name: Updating release notes
+        run: |
+          RELEASE_TAG=$(git describe --tags --abbrev=0)
```
Comment (rcmadhankumar) [inline .github/workflows/release.yml:83]: Release notes contains kapp controller and kctrl content?

Comment (kumaritanushree) [reply .github/workflows/release.yml:83]: fixed

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -42,23 +48,133 @@ jobs:
...
+          KAPP_CONTROLLER_IMAGE=$(yq e '.spec.template.spec.containers[] | select(.name == "secretgen-controller") | .image' release/release.yml)
```
Comment (rcmadhankumar) [inline .github/workflows/release.yml]: The variables referred in this job are still referring to Kapp controller.

Comment (kumaritanushree) [reply .github/workflows/release.yml]: fixed

**Comment by rcmadhankumar** [comment]
LGTM!

**Review by praveenrewar — APPROVED** [review]
LGTM!

**Pull request merged**